packages/strapi-design-system/src/helpers/getFocusableNodes.ts

```diff
@@ -3,9 +3,11 @@
  * The includeNegativeTabIndex aims to provide this capability
  *
  */
-export const getFocusableNodes = (node: HTMLElement, includeNegativeTabIndex: boolean): Element[] => {
+export const getFocusableNodes = (node: HTMLElement, includeNegativeTabIndex?: boolean): HTMLElement[] => {
   const nodes = [
-    ...node.querySelectorAll('a, button, input, textarea, select, details, [tabindex]:not([tabindex="-1"])'),
+    ...node.querySelectorAll<HTMLElement>(
+      'a, button, input, textarea, select, details, [tabindex]:not([tabindex="-1"])',
+    ),
   ];
   const focusables = nodes.filter((node) => {
```

Comment:
is it worth adding a return type of `boolean` here?

Reply:
I'm not sure it's necessary because it doesnt have to be a boolean I guess? What's your reason for thinking about including it? :)